scripts/jenkins/groovy/defineTestStages.groovy

```diff
@@ -87,7 +89,7 @@ def call(final pipelineContext) {
     [
       stageName: 'R3.5 Init Java 8', target: 'test-r-init', rVersion: '3.5.3', javaVersion: 8,
```

Comment:
question: do we still need the java version specification? and r/py? it is specified in the docker image

Reply:
no, once imageSpecifier is present, this info is ignored, but I decided to leave it here since it could be improved further so that the imageSpecifier is not necessary